libs/web3/src/lib/use-gas-price.ts

```diff
@@ -0,0 +1,105 @@
+import { useEffect, useState } from 'react';
+import { useWeb3React } from '@web3-react/core';
+import { useEthereumConfig } from './use-ethereum-config';
+import BigNumber from 'bignumber.js';
+
+const DEFAULT_INTERVAL = 15000; // 15 seconds
+
+export enum ContractMethod {
```

Comment:
**nitpick (non-blocking)**:  Another minor documentation thing, but it would be nice to know where these values come fromm if things change (or other bridges get developed). A comment is probably overkill, but maybe add it to the PR description? 

I assume the easy method is by checking the contract source on Etherscan... 🤷

Reply:
Yup, it's straight outta compt... I mean contract (on etherscan) 😉